specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/putMachineLearningServicesQuotaRequestLowPriority.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "scope": "subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.MachineLearningServices/locations/eastus",
+    "resourceName": "TotalLowPriorityCores",
+    "api-version": "2021-03-15",
+    "createQuotaRequest": {
+      "properties": {
+        "limit": 200,
+        "unit": "Count",
+        "name": {
+          "value": "TotalLowPriorityCores"
+        },
+        "resourceType": "lowPriority"
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.MachineLearningServices/locations/eastus/providers/Microsoft.Quota/serviceLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "name": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "type": "Microsoft.Quota/ServiceLimits",
+        "properties": {
+          "provisioningState": "InProgress",
+          "message": ""
+        }
+      }
+    },
+    "200": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.MachineLearningServices/locations/eastus/providers/Microsoft.Quota/serviceLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "name": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "type": "Microsoft.Quota/serviceLimits",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "message": "Request succeeded",
+          "properties": {
+            "properties": {
+              "limit": 200,
+              "unit": "Count",
+              "currentValue": 160,
+              "name": {
+                "value": "TotalLowPriorityCores",
+                "localizedValue": "Total Cluster Low Priority Regional vCPUs"
+              },
```

Comment:
The same comment regarding the schema mismatch betwen request body and response body. Please fix this.

Reply:
Updated the schema as suggested.